docs/userGuide/contentAuthoring.md

```diff
@@ -607,4 +607,50 @@ MarkBind automatically generates heading anchors for your page.
 
 When the reader hovers over a heading in your page, a small anchor icon <i class="fas fa-anchor"></i> will become visible next to the heading. Clicking this icon will redirect the page to that heading, producing the desired URL in the URL bar that the reader can share with someone else. Try it with the headings on this page!
 
-</div>
\ No newline at end of file
+### Layouts
+
+A layout is a set of styles that can be applied to a page, or glob of pages. It consists of the following files:
+
+- `footer.md` : See [Using Footers](#using-footers)
+- `head.md` : See [Inserting content into a page's head element](#inserting-content-into-a-pages-head-element)
+- `navigation.md` : See [Site Navigation](#site-navigation)
+- `styles.css` : Contains custom styles
+- `scripts.js` : Contains custom javascript
+
+These files will be automatically appended to a page upon generation, allowing you to quickly apply styles to a batch of pages at once.

```

Comment:
Seems to be repeating the same thing as `The default layout is automatically applied to every single page.` below?

Or do you mean it is generated when doing a `markbind init`?

Reply:
I meant to advertise the use case, like the purpose of templates is to quickly apply stuff to a batch of files.

I moved the sentence up so it's a little clearer.